sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-pre-validation-service-models.ts

```diff
@@ -5,6 +5,49 @@ import {
 import { ProcessSummary } from "@sims/utilities/logger";
 import { EntityManager } from "typeorm";
 
+const ACCEPT_ASSESSMENT_BLOCKING_VALIDATIONS = [
+  ECertFailedValidation.DisabilityStatusNotConfirmed,
+  ECertFailedValidation.MSFAACanceled,
+  ECertFailedValidation.MSFAANotSigned,
+  ECertFailedValidation.HasStopDisbursementRestriction,
+  ECertFailedValidation.NoEstimatedAwardAmounts,
+];
```

Comment:
Not part of the story and PR. 

Why is invalid SIN not considered here?? e.g. a temporary SIN which is past the expiry date.

Reply:
This is the combination of the previous requirements from #2706 and the current one related to `ECertFailedValidation.NoEstimatedAwardAmounts`.
These are the ones identified by the business as meaningful to prevent the user from accepting the assessment.
The SIN would possibly prevent the student from even submitting the application.
If there is something not following the above-mentioned requirement please let me know and we should continue the conversation here. 
If the questions are outside the scope of the above-mentioned requirements lets continue the conversation outside the PR 😉

Reply:
If the questions are outside the scope of the above-mentioned requirements lets continue the conversation outside the PR - Sure.